Gem/Code/Source/Components/UI/UiSettingsComponent.h

```diff
@@ -24,6 +25,7 @@ namespace MultiplayerSample
 
     class UiSettingsComponent
```

Comment:
Would love a class on all public classes ;)

Reply:
Added a comment, lmk if there's anything else you'd like it to cover.